### PR TITLE
feat: implement <font place="InactiveWindow"> (ref scope: 2.4.7)

### PR DESCRIPTION
**Pull request by ludg1e**
Thanks you @Consolatis for the help of how to implement it.

Is the scope repo automatically updated or should I do it?

It works:
image

### CODE REVIEW

File: docs/rc.xml.all [hunk 1]
```diff
@@ -25,6 +25,12 @@
       <slant>normal</slant>
       <weight>normal</weight>
     </font>
+    <font place="InactiveWindow">
+      <name>sans</name>
+      <size>10</size>
+      <slant>italic</slant>
+      <weight>normal</weight>
+    </font>
```
Comment (Consolatis) [inline docs/rc.xml.all:33]: Suggest reverting `slant` to `normal` here to a) match the hardcoded default in `src/config/rcxml.c` and b) to not surprise users by the sudden change in defaults.

**Comment by Consolatis** [comment]
Apart from the `slant` default in the example config this looks great, thanks :)

> Is the scope repo automatically updated or should I do it?

No, that one is updated manually. Feel free to send a PR there or just wait for @johanmalm to update it later.

File: docs/rc.xml.all (hunk 1 above)
Comment (ludg1e) [reply docs/rc.xml.all:33]: Done

Comment (ludg1e) [reply docs/rc.xml.all]: Fixed

**Comment by Consolatis** [comment]
:)

**Pull request merged**